src/util/vocabularyChecker/detectWrongSyntax.ts

```diff
@@ -20,7 +20,7 @@ export interface Position { fromIndex: number; toIndex: number; }
  * @param text the user-input
  */
 export function collectInvalidWordsInIssues(text: string): void {
-	text = text.replace(/[.,\/#!$%\^&\*;:{}=\-_`~()]/g, "");
+	text = text.replace(/[.,\/#!$%\^&\*;:{}=\-_`~()]/g, " ");
```

Comment:
Better create a copy of the initial text so the inital text will not be accidentally changed.